### ISSUE TITLE
Help with relation

### ISSUE THREAD

**Issue by rbruhn** [laravel-auto-presenter/laravel-auto-presenter#100]
I use Auth::user()->profile->avatar_small to display the user avatar in the navigation menu. I created a presenter for the Profile model but it doesn't seem to work. Was wondering what I might be missing.

I have the getPresenter method in Profile Model
```
public function getPresenterClass()
{
    return ProfilePresenter::class;
}
```
My ProfilePresenter
```
class ProfilePresenter extends BasePresenter
{
    /**
     * Build link to avatar if it exists. Otherwise set default
     * @return string
     */
    public function avatar_small() {
        dd('test');
        $avatar = $this->wrappedObject->avatar;

        return $avatar->exists() ? $avatar->url('small') : url('avatars/small/missing.png');
    }
}
```
I use Paperclip for file uploads. However, doesn't matter as the "test" dump is never even reached.
Edit: I figured putting the auth user in a view composer and loading the profile would help with this but it didn't.

**Comment by lguima** [comment]
Have you saw the **Troubleshooting** section on `README.md`?

> ## Troubleshooting
> 
> If an object isn't being decorated correctly in the view then there's a good chance that it's simply not in existence when the view begins to render. For example, lazily-loaded relationships won't be decorated. You can fix this by eager-loading them instead. Auth::user() will never be decorated. I prefer to bind $currentUser to my views, anyway.
> 
> If an object is a relation of another object and it isn't being decorated in the view, you might not have added the `Decoratable` interface to the other object. To fix this, add the `Decoratable` interface to the other object.

Reading again I figured out that this answer [my issue](https://github.com/laravel-auto-presenter/laravel-auto-presenter/issues/98) too.

**Comment by rbruhn** [comment]
Alright... I found I needed to also have a User presenter. I thought this was fixed with #91 but apparently not.

**Comment by rbruhn** [comment]
@lguima Thanks. Already figured it out after reading more.
